Misc/NEWS.d/next/Library/2024-07-21-10-45-24.gh-issue-122081.dNrYMq.rst

```diff
@@ -0,0 +1 @@
+Fixed crash in optional :func:`!decimal.IEEEContext()` function.
```

Comment:
We should also document `decimal.IEEEContext` as well but in another PR.

Reply:
here it's: https://github.com/python/cpython/pull/122003